NEWS.md

```diff
@@ -3,6 +3,8 @@
 ## Changes to plotly.js
 
 * This version of the R package upgrades the version of the underlying plotly.js library from v2.5.1 to v2.11.1. This includes many bug fixes and improvements. The [plotly.js release page](https://github.com/plotly/plotly.js/releases) has the full list of changes.
+* raster2uri() supports nativeRaster objects. This enables nativeRaster support for
+  the annotation_raster() geom (#2174, @zeehio).
```

Comment:
Interesting, thanks! Could you move this bullet down to the `New features` section?

Reply:
Done!